src/hotspot/share/utilities/elfFile.cpp

```diff
@@ -1625,6 +1626,20 @@ bool DwarfFile::LineNumberProgram::get_filename_from_header(const uint32_t file_
   return false;
 }
 
+// Remove everything before the last slash including the slash itself to get the actual filename. This is required, for
+// example, for Clang debug builds which emit a relative path while GCC only emits the filename.
+void DwarfFile::LineNumberProgram::strip_path_prefix(char* filename, const size_t filename_len) {
+  char* last_slash = strrchr(filename, '/');
```

Comment:
Maybe use `os::file_separator()` instead of `/` here.

Reply:
Also corrected a second usage of `/` in the DWARF parser code.